Storage/Storage/CoreData/CoreDataIterativeMigrator.swift

```diff
@@ -144,20 +144,34 @@ private extension CoreDataIterativeMigrator {
         return tempDestinationURL
     }
 
-    /// Move the original source store to a backup location.
+    /// Move the original source store files to a backup location (typically /Documents/backup).
     ///
-    func makeBackup(at storeURL: URL) throws -> URL {
-        let backupURL = storeURL.deletingLastPathComponent().appendingPathComponent("backup")
-        try? fileManager.removeItem(at: backupURL)
-        try? fileManager.createDirectory(atPath: backupURL.path, withIntermediateDirectories: false, attributes: nil)
+    /// The files that will be moved are:
+    ///
+    /// - {store_filename}.sqlite
+    /// - {store_filename}.sqlite-wal
+    /// - {store_filename}.sqlite-shm
+    ///
+    /// Where {store_filename} is most probably "WooCommerce".
+    ///
+    func moveStoreFilesToBackupFolder(storeURL: URL) throws -> URL {
+        let storeFolderURL = storeURL.deletingLastPathComponent()
+        let backupURL = storeFolderURL.appendingPathComponent("backup")
+
+        if fileManager.fileExists(atPath: backupURL.path) {
+            try fileManager.removeItem(at: backupURL)
+        }
+        try fileManager.createDirectory(at: backupURL, withIntermediateDirectories: false, attributes: nil)
+
         do {
-            let files = try fileManager.contentsOfDirectory(atPath: storeURL.deletingLastPathComponent().path)
-            try files.forEach { (file) in
-                if file.hasPrefix(storeURL.lastPathComponent) {
-                    let fullPath = storeURL.deletingLastPathComponent().appendingPathComponent(file).path
-                    let toPath = URL(fileURLWithPath: backupURL.path).appendingPathComponent(file).path
-                    try fileManager.moveItem(atPath: fullPath, toPath: toPath)
-                }
+            try fileManager.contentsOfDirectory(atPath: storeFolderURL.path).map { fileName in
+                storeFolderURL.appendingPathComponent(fileName)
+            }.filter { fileURL in
+                // Only include files that have the same filename as the store (sqlite) filename.
+                fileURL.deletingPathExtension() == storeURL.deletingPathExtension()
+            }.forEach { fileURL in
+                let toPath = backupURL.appendingPathComponent(fileURL.lastPathComponent).path
+                try fileManager.moveItem(atPath: fileURL.path, toPath: toPath)
```

Comment:
nit: could we also use `URL` instead of path string here? In the [Apple documentation on `FileManager`](https://developer.apple.com/documentation/foundation/filemanager):

> The use of the NSURL class is generally preferred for specifying file-system items because URLs can convert path information to a more efficient representation internally. 

```suggestion
                let toURL = backupURL.appendingPathComponent(fileURL.lastPathComponent)
                try fileManager.moveItem(at: fileURL, to: toURL)
```

and adding this to `FileManagerProtocol`:

```swift
func moveItem(at srcURL: URL, to dstURL: URL) throws
```

Reply:
Thank you! I definitely missed that we're using `FileManagerProtocol` and to think I was the one who added it in `IterativeMigrator`. 😄 

Fixed in b6d2b2121290b76bf6c8b1c633fbfbce04972524 and 9d7b883702bc216c0ced62a7b3bcc727881a848f.